### DISCUSSION TITLE
Trash/Recycle Bin

### DISCUSSION THREAD

**Discussion by HubKing** [tsujan/FeatherNotes#358] [Ideas]
How about adding an option to use trash Instead of deleting with a confirmation ("it can't be undone!")? It basically is just creating a special "Node", and when the user selects "Delete Node", moving that node into that special "Trash" node with some meta data (original location, deletion time) without confirmation, so that the user could "undo" the deletion.

**Comment by tsujan** [comment]
It's a good idea, and I might consider it if I had the time, but the many projects that I maintain — my apps and LXQt  — leave me no free time.
